plugins/window-state/src/lib.rs

```diff
@@ -342,9 +351,15 @@ impl Builder {
         self
     }
 
+    pub fn map_label(mut self, map_fn: fn(&str) -> &str) -> Self {
```

Comment:
Let's use a generic instead of a function pointer to allow using closures
```suggestion
    pub fn map_label<F>(mut self, map_fn: F) -> Self 
    where
            F: Fn(&str) -> &str + Sync + Send,
    {
```